lib/date.js

```diff
@@ -2,7 +2,7 @@ import { truncate } from './helpers'
 
 export default function inspectDate(dateObject, options) {
   // If we need to - truncate the time portion, but never the date
-  const split = new Date(dateObject).toJSON().split('T')
+  const split = dateObject.toJSON().split('T')
```

Comment:
Nice 👍

Reply:
Yeah, the funniest thing that didn't work only in IE )